masonite/managers/Manager.py

```diff
@@ -1,12 +1,14 @@
 class Manager(object):
-
+    """
+    Manager container class
+    """
     def __init__(self, container=None):
```

Comment:
Let's put a space between all class docstrings and the first class method. So just put a space between line 4 and 5